Jenkinsfile

```diff
@@ -0,0 +1,41 @@
+pipeline {
+  agent {
+    label 'x86&&macOS&&Apps'
+  }
+  environment {
+    VIEW = 'dsp'
+    REPO = 'lib_dsp'
+  }
+  options {
+    skipDefaultCheckout()
+  }
+  stages {
+    stage('Get view') {
+      steps {
+        prepareAppsSandbox("${VIEW}", "${REPO}")
+      }
+    }
+    stage('Library checks') {
+      steps {
+        libraryChecks("${REPO}")
+      }
+    }
+    stage('App Notes') {
+      steps {
+        buildAllApps("${REPO}/examples")
+        buildAllAppNotes("${REPO}/examples")
```

Comment:
You could call `buildAppNote("${REPO}/AN00209_xCORE-200_DSP_Library")` whilst the app note is located in the root directory.

Reply:
As https://github.com/xmos/xmos_jenkins_shared_library/pull/56 has now been merged you will need to call `xcoreAllAppNotesBuild()`, if the above change is not made instead.